charts/agent/templates/_helpers.tpl

```diff
@@ -506,3 +506,8 @@ sysdig_capture_enabled: false
 true
 {{- end }}
 {{- end }}
+
+{{/* Return the name of the local forwarder configmap */}}
+{{- define "agent.localForwarderConfigMapName" }}
+{{- include "agent.configmapName" . | printf "%s-local-forwarder" | trunc 63 | trimSuffix "-" }}
```

Comment:
If the agent configmapName is loger we will cut out the `local-forwarder` suffix creating clash, I suggest to cut the previous part and then append the suffix, like:
```suggestion
{{- include "agent.configmapName" . | trunc 46 | trimSuffix "-" | printf "%s-local-forwarder" }}
```